components/TabBar/selectors.js

```diff
@@ -89,6 +92,10 @@ export const getVisibleTabs = () => ([
     type: TAB_CART,
     label: 'tab_bar.cart',
   },
+  {
```

Comment:
I guess this should only be shown when feature flag allows it, right?

Reply:
Ups. Fixed.